Add 2025-04-30 TAC meeting minutes (#77)
PQCA/TAC: meeting-minutes/2025/2025-04-30 TAC.md

## **Voting Representative Attendance (*Alphabetical by 1st name*)**

### **Premier Member Representatives**

*  Brian Jarvis, Amazon Web Services Inc. \[TAC Chair\]  
*  Nicklas Körtge (IBM Research Zurich)  
*  Norman Ashley, Cisco  
*  Sophie Schmieg, Google (Not attended)  
* Yarkin Doroz, NVIDIA

### **Project Representatives**

*  Nigel Jones, IBM \[PQCP\]  
*  Spencer Wilson, U Waterloo \[OQS\]

## **Non-Voting Representative Attendance**

*  Joe Livingston (IBM)

### **LF Staff**

*  Hart Montgomery  
*  Balaji Ethirajulu

### **Other Attendees**

* Aditya Koranga (CORAN Labs) (TAC Vice chair)  

> * Matvii Kistaiev (IntellectEU) \- Not attended  

Reviewer: Curious about this one, an "other attendee" but "not attended"? I'm confused.

* Miguel Diaz Gonçalves (Portugal customs & Tax authority)  
* Angel Camacho

---

## **Introduction of 1st time attendees**

*  Miguel Diaz Gonçalves (Portugal customs & Tax authority) \- Joining from Portugal.

---

## **Announcements**

---

## **TAC rep to outreach committee and Blog \- Balaji Ethirajulu**

Balaji Ethirajulu, as discussed in the board meeting, suggested having a TAC member to participate in the outreach committee to form a bridge between technical and marketing committee. TAC representatives can talk about technology topics and gather key takeaways from the marketing committee and bring it to the technical committee.

Yarkin suggested that Nicholas Harrigan to join the marketing committee. Ethirajulu suggested if Harrigan can also attend TAC meeting and provide marketing updates to TAC. Yarkin will setup a meeting with Harrigan to discuss the possibilities of engagement.

> Ethirajulu also suggested that TAC members and other PQCA members can write blogs on different topics around PQCA. He mentioned that he will create an excel sheet with initial suggested topics and share it with TAC members, so they can decide and come up with various blog topics

Reviewer: We could add the link you shared with the spreadsheet: https://docs.google.com/spreadsheets/d/13nBoOJQQ109OJ0RiYAOXg7OW1Fhj68F_OAIrNnQvBeY

> Ethirajulu also suggested that TAC members and other PQCA members can write blogs on different topics around PQCA. He mentioned that he will create an excel sheet with initial suggested topics and share it with TAC members, so they can decide and come up with various blog topics

Reviewer: ```suggestion
Ethirajulu also suggested that TAC members and other PQCA members can write blogs on different topics around PQCA. He created [this spreadsheet](https://docs.google.com/spreadsheets/d/13nBoOJQQ109OJ0RiYAOXg7OW1Fhj68F_OAIrNnQvBeY) with initial suggested topics and shared it with TAC members, so they can decide and come up with various blog topics.
```

We could add the link you shared with the spreadsheet (see suggestion above).

Brian Jarvis also stated that from the retrospective discussions, lots of interest shown by the TAC members regarding marketing of PQCA.  

---

## **OQS Update \-Spencer Wilson**

> * Shipped 13.0 Lipos release. This is the first release that includes MLKEM native code. Improved some of the onramp signatures on the experimental side. Added some features for seed handling for MLKEM.  

Reviewer 2: ```suggestion
* Shipped 13.0 liboqs release. This is the first release that includes MLKEM native code. Improved some of the onramp signatures on the experimental side. Added some features for seed handling for MLKEM.  
```

> * Launched our community survey as part of our presentation at ICMC. Reviewed initial results of the survey in the TSC. According to initial results of the survey, there are a number of people using the OQS, but not participating in the meeting and not involved in GitHub. They might be interested in joining us. We can reach out to them and hopefully they can join us and contribute to the code base.

Reviewer 2: ```suggestion
* Launched the OQS community survey as part of a presentation at ICMC. Reviewed initial results of the survey in the TSC. According to initial results of the survey, there are a number of people using OQS, but not participating in meetings and not involved in GitHub. They might be interested in joining us. We can reach out to them and hopefully they can join us and contribute to the OQS code base.
```
I think it's important to note that the survey was focused on OQS and not PQCA as a whole. In particular, the user survey didn't mention non-OQS PQCA software.
